Resources/public/js/views/fields/ez-image-editview.js

```diff
@@ -51,6 +51,55 @@ YUI.add('ez-image-editview', function (Y) {
         },
 
         /**
+         * Set the file attribute with a struct based on the file object from the
+         * input file element. The struct uri is also tested by a test image to see
+         * if it is valid
+         *
+         * @method _base64ToFileStruct
+         * @protected
+         * @param {EventFacade} e event facade
+         * @param {File} file the File object from the input file element
+         */
+        _base64ToFileStruct: function (file, e) {
+            var base64 = e.target.result.replace(/^.*;base64,/, ''),
+                myImage = new Image(100, 200),
+                fileStruct = this._createFileStruct(file, base64);
+
+            myImage.src = fileStruct.displayUri;
+            myImage.addEventListener ("error", Y.bind(function() {
+                this._removeImageUpdate();
+                this._setWarningNotAnImage(file);
+
+            },this));
+            myImage.addEventListener ("load", Y.bind(function() {
+                this._set('file', fileStruct);
+            }, this));
+
+            e.target.onload = undefined;
+        },
+
+        /**
+         * Removes the image update class
+         *
+         * @method _removeImageUpdate
+         * @protected
+         */
+        _removeImageUpdate: function () {
```

Comment:
bad name, what is "Image Update" ? should be called `_removeImageBeingUpdatedClass` (and the doc comment should be fixed accordingly)